rsocket/internal/Common.cpp

```diff
@@ -92,6 +92,30 @@ StreamInterruptedException::StreamInterruptedException(int _terminatingSignal)
 
 ResumeIdentificationToken::ResumeIdentificationToken() {}
 
+ResumeIdentificationToken::ResumeIdentificationToken(const std::string& token) {
+  auto getNibble = [&token](size_t i) {
+    uint8_t nibble;
+    if (token[i] >= '0' && token[i] <= '9') {
```

Comment:
Can you use `strtol()` here or `std::string::stoi()` here?

Reply:
`std::strtol()`'s parameters make it harder to convert only one char

`std::string::stoi()` doesn't get compiled with our setup, likely because of
https://stackoverflow.com/questions/19285775/why-are-stdstoi-and-stdarray-not-compiling-with-g-c11
It also has similar parameters like `std::strtol()`

Reply:
Isn't it just `strtol(token.data() + i, token.data() + i + 1, 16)`?

That `stoi()` thing is weird, never seen that before...

Reply:
Unfortunately no,  
```
*endptr*
Reference to an object of type char*, whose value is set by the function to the next character in str after the numerical value.
This parameter can also be a null pointer, in which case it is not used.
```

Reply:
*blows raspberry*